test_cind/model.py

```diff
@@ -0,0 +1,36 @@
+import torch
+import numpy as np
+from os import path
+from torch import nn
+
+save_dir = '/data/ojw/onnx2tf/test_cind/'
```

Comment:
It is a bit risky to show the root hierarchy as a sample. `'/data/ojw/onnx2tf/test_cind/'`
How about this? `'data/ojw/onnx2tf/test_cind/'`